spec/cypress/integration/Cookies.feature

```diff
@@ -0,0 +1,40 @@
+Feature: Cookie page
+  Everybody wants to set cookie preferences
+
+  Scenario: Setting cookie preferences on cookie page
+    Given I am on "cookie" page
+    Then "cookie consent" radios are unchecked
+    When I set "cookie consent" radio to "on"
+    And I click the submit button
+    Then cookie preferences have changed
+    And "cookie consent" radio with value "on" is checked
+    When I set "cookie consent" radio to "off"
+    And I click the submit button
+    Then cookie preferences have changed
+    And "cookie consent" radio with value "off" is checked
```

Comment:
I would love to use indentation for signifying `givens`, `whens` and `thens` - so one more indent means one more step in the logical structure. But that might be a pipe dream.

Reply:
I'm aware it usually follows that pattern but that's not how Cypress works!

How would you usually do a multi-page tests with assertions on both pages with cucumber?

Reply:
Have been doing some reading and following a pattern of given->when->then->when->then isn't that unusual - but having a then before the first when definitely is

Reply:
Definitely would use more And's and less main statements - the main criteria I guess would be readability. Can we group couple of GIVEN, WHEN , THEN statements so that we run the scenario - for example GIVEN "Scenario foo had run" , WHEN I click a button THEN "North Korea sends an angry tweet to trump"

Reply:
I'm not sure I understand what you're asking - but I've added some whitespace, has that helped readability?

Reply:
Just try to avoid Then -> When combinations

`Given I am on "cookie" page
    Then "cookie consent" radios are unchecked
    When I set "cookie consent" radio to "on"`